ot3_state_manager/ot3_state_manager/util.py

```diff
@@ -3,28 +3,92 @@
 from __future__ import annotations
 
 import enum
-import hashlib
+from enum import Enum, unique
 
+from opentrons.hardware_control.types import OT3Axis
 
-class Direction(enum.Enum):
+
+class Direction(int, enum.Enum):
     """Class representing direction to move axis in."""
 
-    POSITIVE = "+"
-    NEGATIVE = "-"
+    POSITIVE = 1
+    NEGATIVE = 0
+
+    @classmethod
+    def from_int(cls, direction_val: int) -> Direction:
+        """Parse integer into direction."""
+        if direction_val == Direction.NEGATIVE:
+            direction = Direction.NEGATIVE
+        elif direction_val == Direction.POSITIVE:
+            direction = Direction.POSITIVE
+        else:
+            raise ValueError(
+                "Value for direction must be either 0 (Negative) or 1 "
+                f"(Positive). You passed {direction_val}."
+            )
+        return direction
+
+
+class SyncPinState(int, enum.Enum):
+    """Class representing sync pin state."""
+
+    HIGH = 1
+    LOW = 0
 
     @classmethod
-    def from_symbol(cls, symbol: str) -> "Direction":
-        """Parse plus or minus symbol into direction."""
-        if symbol == "+":
-            return cls.POSITIVE
-        elif symbol == "-":
-            return cls.NEGATIVE
+    def from_int(cls, state_val: int) -> SyncPinState:
+        """Parse integer into sync pin state."""
+        if state_val == 0:
+            state = SyncPinState.LOW
+        elif state_val == 1:
+            state = SyncPinState.HIGH
         else:
             raise ValueError(
-                f'Symbol "{symbol}" is not valid.' f' Valid symbols are "+" and "-"'
+                "Value for state must either be 0 (LOW) or 1 (HIGH). "
+                f"You passed {state_val}."
             )
 
+        return state
 
-def get_md5_hash(data: bytes) -> bytes:
-    """Convert bytes into a md5 hash."""
-    return hashlib.md5(data).hexdigest().encode()
+
+@unique
+class MoveMessageHardware(Enum):
+    """Enum representing a mapping of an integer hardware id to an OT3Axis object."""
+
+    def __new__(cls, hw_id: int, axis: OT3Axis) -> MoveMessageHardware:
```

Comment:
can we do this in a way that's easier to typecheck? perhaps as two objects, or with helper functions?

Reply:
Removed `__new__` method and added helper functions to lookup by id.
26c23bb154ed560b05060954bc2d0f4c682ec017